src/gfx/context.h

```diff
@@ -10,6 +10,7 @@ typedef struct GFX_Context GFX_Context;
 
 void GFX_Context_Attach(HWND hwnd);
 void GFX_Context_Detach();
+void GFX_Context_SetVsync(bool vsync);
```

Comment:
In another place we say `VSync` but here we use `Vsync`, let's make it `VSync` here as well.